package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "workspaces": [
     "packages/client",
+    "packages/common",
```

Comment:
I created a new package, common, which can be imported in both client and main using `import whatever from "poodle-common/whatever`.